modules/editable-layers/src/edit-modes/draw-90degree-polygon-mode.ts

```diff
@@ -25,7 +25,7 @@ export class Draw90DegreePolygonMode extends GeoJsonEditMode {
     const {mapCoords} = props.lastPointerMoveEvent;
 
     let p3;
-    if (clickSequence.length === 1) {
+    if (clickSequence.length <= 1) {
```

Comment:
first two clicks make a line, then the bearing etc. can be calculated form that.